lib/teleterm/clusters/cluster_servers.go

```diff
@@ -59,18 +71,8 @@ func (c *Cluster) GetServers(ctx context.Context, r *api.GetServersRequest) (*Ge
 			return trace.Wrap(err)
 		}
 		defer authClient.Close()
-		sortBy := types.GetSortByFromString(r.SortBy)
-
-		resp, err = authClient.ListResources(ctx, proto.ListResourcesRequest{
-			Namespace:           defaults.Namespace,
-			ResourceType:        types.KindNode,
-			Limit:               r.Limit,
-			SortBy:              sortBy,
-			StartKey:            r.StartKey,
-			PredicateExpression: r.Query,
-			SearchKeywords:      client.ParseSearchKeywords(r.Search, ' '),
-			UseSearchAsRoles:    r.SearchAsRoles == "yes",
-		})
+
+		servers, err = apiclient.GetAllResources[types.Server](ctx, authClient, req)
```

Comment:
I'm pretty sure `Cluster.GetServers` should not use `apiclient.GetAllResources`. This method is used to display the paginated table in the cluster view in Connect, so we want to fetch only one table at a time.

I was about to ask if there's a specific reason behind changing just `Cluster.GetServers` but not `Cluster.GetDatabases` or `Cluster.GetKubes`, but it seems like all three places should continue using `authClient.ListResources` (or be refactored to use `apiclient.GetResourcePage`) as they're all responsible for feeding data for those resource tables.